docs/blog/lion-without-polyfills.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Lion without poylfills
+published: true
+description: Lion has been a long user of the scoped registry - always requiring a polyfill - but no more. Load the polyfill only if you need it.
+date: 2022-04-04
+tags: [javascript, polyfills]
+cover_image: /blog/images/introducing-lions-website-cover-image.jpg
+---
+
+From the start we have been focused on making it very explicit which sub component are used within a given custom element.
+This is commonly revered to as [component composition](https://lit.dev/docs/composition/component-composition/).
+On top of that we scoped these [sub components](https://open-wc.org/docs/development/scoped-elements/) to the [current shadow root](https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Scoped-Custom-Element-Registries.md).
+
+This means that you can use a lion component like `lion-listbox` and never have to worry about if the internally used components clash with others you are already using.
```

Comment:
Maybe we should mention here that `lion-listbox` uses LionOptions internally as 'lion-options'/

Even better: give a more hypothetical example, since most of our internal compositions are workarounds for the to be implemented [a11y spec](https://wicg.github.io/aom/notification-api.html).

So the most simple and understandable use case for a consumer would be shadow dom usage like `MyCard/my-card` that uses `MyCardHeader/my-card-header`

Reply:
the example below is for Listbox? 🤔

not clear enough? or worth repeating here?

Reply:
I would go with a purely hypothetical one consumers could immediately relate to without knowing our codebase and without the complexity of needing to be rendered to light dom (apart from the fact that inside our codebase it's implemented way different).
(MyLionCard and MyLionCardHeader would have a really low 'cognitive load')